WooCommerce/Classes/POS/TabBar/POSIneligibleView.swift

```
@@ -2,129 +2,259 @@ import SwiftUI
+                                isLoading = true
+                                try await onRefresh()
+                                isLoading = false
+                            } catch {
+                                // TODO: WOOMOB-720 - handle error if needed, e.g., show an error message
+                                print("Error refreshing eligibility: \(error)")
+                                isLoading = false
+                            }
                         }
+                    } label: {
+                        Text(Localization.refreshEligibility)
                     }
-                } label: {
-                    Text(Localization.refreshEligibility)
+                    .buttonStyle(POSFilledButtonStyle(size: .normal, isLoading: isLoading))
+
+                    Button {
+                        dismiss()
+                    } label: {
+                        Text(Localization.dismiss)
+                    }
+                    .buttonStyle(POSOutlinedButtonStyle(size: .normal))
+                }
+                .containerRelativeFrame(.horizontal) { length, _ in
+                    length * 0.5 - 132
                 }
-                .buttonStyle(POSFilledButtonStyle(size: .normal, isLoading: isLoading))
             }
 
             Spacer()
         }
         .padding(POSPadding.large)
     }
 
-    private var reasonText: String {
+    private var suggestionText: String {
         switch reason {
         case .notTablet:
-            return NSLocalizedString("pos.ineligible.reason.notTablet",
-                                     value: "POS is only available on iPad.",
-                                     comment: "Ineligible reason: not a tablet")
+            return NSLocalizedString("pos.ineligible.suggestion.notTablet",
+                                     value: "Please use a tablet to access POS features.",
+                                     comment: "Suggestion for not tablet: use iPad")
```

Should the tab show up on the phone? I can't see it.

---

Not now and in the near future, but it's an eligibility check that I thought it's better to throw an ineligible error than `fatalError` or no-op. I can separate the ineligible reason enum into two, one for visibility and the other for eligibility so that this case won't have to be handled. Created an issue in WOOMOB-756.